### PR TITLE
use `base_model` to encode that a model is a quantized version of another model

### PR DESCRIPTION
**Pull request by julien-c**
## screenshots

base_model_quantized
base_model_quantized-dark

## Credits

idea from @PrithivirajDamodaran

### CODE REVIEW

**Review by davanstrien — APPROVED** [review]
LGTM!

**Comment by julien-c** [comment]
re-requesting review @davanstrien as i added some screenshots to the doc page (maybe it's too much?)

File: docs/hub/model-cards.md [hunk 1]
```diff
@@ -123,6 +123,27 @@ This metadata will be used to display the base model on the model page. Users ca
...
+<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/base_model_adapter.png"/>
+<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/base_model_adapter-dark.png"/>
+</div>
+
+For a quantized version of another model:
+
+<div class="flex justify-center">
+<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/base_model_quantized.png"/>
+<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/base_model_quantized-dark.png"/>
+</div>
```
Comment (osanseviero) [inline docs/hub/model-cards.md:145]: No need for so many images imo, just showing fine-tuned and merge might be fine (for merge I would also show a yaml example with two base models)

**Review by osanseviero — APPROVED** [review]
Nice!

File: docs/hub/model-cards.md [hunk 2]
```diff
@@ -110,7 +110,7 @@ If it's not specified, the Hub will try to automatically detect the library type
 
 ### Specifying a base model
 
-If your model is a fine-tune or adapter of a base model, you can specify the base model in the model card metadata section. This information can also be used to indicate if your model is a merge of multiple existing models. The `base_model` field can either be a single model ID, or a list of one or more base_models (specified by their Hub identifier). 
+If your model is a fine-tune, an adapter, or a quantized version of a base model, you can specify the base model in the model card metadata section. This information can also be used to indicate if your model is a merge of multiple existing models. The `base_model` field can either be a single model ID, or a list of one or more base_models (specified by their Hub identifier).
```
Comment (pcuenca) [inline docs/hub/model-cards.md:113]:
```suggestion
If your model is a fine-tune, an adapter, or a quantized version of a base model, you can specify the base model in the model card metadata section. This information can also be used to indicate if your model is a merge of multiple existing models. The `base_model` field can either be a single model ID, or a list of one or more base_models (specified by their Hub identifiers). 
```

File: docs/hub/model-cards.md (hunk 1 above)
Comment (julien-c) [reply docs/hub/model-cards.md:145]: quantized is important too no?

Comment (julien-c) [reply docs/hub/model-cards.md:145]: let me add the yaml example with two base models

**Comment by julien-c** [comment]
Thanks for your reviews, merging!

**Comment by julien-c** [comment]
[see the page in prod here](https://huggingface.co/docs/hub/model-cards#specifying-a-base-model)

**Pull request merged**